lm_eval/tasks/__init__.py

```diff
@@ -436,6 +440,43 @@ def _get_task_and_group(self, task_dir: str):
         :return
             Dictionary of task names as key and task metadata
         """
+
+        def _populate_tags_and_groups(config, task, tasks_and_groups, print_info):
+            # TODO: remove group in next release
+            for attr in ["tag", "group"]:
+                if attr in config:
```

Comment:
these conditionals may be combined? 
`if attr in config and attr == "group" and print_info:`